mappings/net/minecraft/util/math/Vec2f.mapping

```diff
@@ -1,7 +1,16 @@
 CLASS net/minecraft/class_241 net/minecraft/util/math/Vec2f
+	FIELD field_1335 SOUTH_EAST_UNIT Lnet/minecraft/class_241;
+	FIELD field_1336 SOUTH_UNIT Lnet/minecraft/class_241;
+	FIELD field_1337 MAX_SOUTH_EAST Lnet/minecraft/class_241;
+	FIELD field_1338 WEST_UNIT Lnet/minecraft/class_241;
+	FIELD field_1339 MIN_SOUTH_EAST Lnet/minecraft/class_241;
```

Comment:
Is this `MIN_NORTH_WEST`?

Reply:
It’s the vector with (Float.MIN_VALUE, Float.MIN_VALUE)

Reply:
So I assume that makes it `MIN_SOUTH_EAST`